File: calculadora.java
import java.util.Scanner;

public class Calculator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Selecione uma operação:");
        System.out.println("1. Adição");
        System.out.println("2. Subtração");

        int choice = scanner.nextInt();

        System.out.print("Digite o primeiro número: ");
        double num1 = scanner.nextDouble();

        System.out.print("Digite o segundo número: ");
        double num2 = scanner.nextDouble();

        double result = 0;

        switch (choice) {
            case 1:
                result = num1 + num2;
                break;
            case 2:
                result = num1 - num2;
                break;
            default:
                System.out.println("Operação inválida.");
                break;
        }

        System.out.println("Resultado: " + result);
    }
}

File: calculadora.py
def add(x, y):
    return x + y

def subtract(x, y):
    return x - y

if __name__ == "__main__":
    while True:
        print("Selecione uma operação:")
        print("1. Adição")
        print("2. Subtração")
        print("3. Sair")

        choice = input("Digite 1, 2 ou 3: ")

        if choice == '3':
            break

        num1 = float(input("Digite o primeiro número: "))
        num2 = float(input("Digite o segundo número: "))

        if choice == '1':
            print("Resultado:", add(num1, num2))
        elif choice == '2':
            print("Resultado:", subtract(num1, num2))